lib/src/signed_video_internal.h

```diff
@@ -211,10 +214,12 @@ struct _gop_info_t {
   size_t hash_list_size;  // The allowed size of the |hash_list|. This can be less than allocated.
   int list_idx;  // Pointing to next available slot in the |hash_list|. If something has gone wrong,
   // like exceeding available memory, |list_idx| = -1.
+  int nalu_list_idx;
   uint8_t gop_hash_init;  // The initialization value for the |gop_hash|.
   uint8_t *nalu_hash;  // Pointing to the memory slot of the NALU hash in |hashes|.
   uint8_t document_hash[MAX_HASH_SIZE];  // Memory for storing the document hash to be signed
   // when SV_AUTHENTICITY_LEVEL_FRAME.
+  uint8_t hash_of_nalu_hash_list[MAX_HASH_SIZE];  // Hash of NALU hashes in GOP.
```

Comment:
Since you have a member named `received_gop_hash` how about naming this `computed_gop_hash`?